packages/core/src/donut-chart.ts

```diff
@@ -122,12 +127,50 @@ export class DonutChart extends PieChart {
 		super.update();
 
 		if (this.center) {
-			const sumOfDatapoints = this.displayData.datasets[0].data.reduce((accum, currVal) => accum + currVal.value, 0);
+			// Set donut center configs
+			this.setCenterConfigs();
 
-			this.center.configs.number = sumOfDatapoints;
+			// Update donut center
 			this.center.update();
 		}
 	}
+
+	getSuppliedCenterConfigs() {
+		// TODO 1.0 - Remove deprecated API
+		const { center, centerLabel, centerNumber } = this.options;
+		const label = center ? center.label : centerLabel;
+		let number = center ? center.number : centerNumber;
+
+		// TODO 1.0 - Remove deprecated API
+		// Warn developer about deprecation
+		if (centerLabel || centerNumber) {
+			console.warn(
+				"`centerLabel` & `centerNumber` are deprecated and will be removed in v1.0, you should switch to",
+				{
+					center: {
+						label: "test",
```

Comment:
It would be cool if we created their actual replacement code here, what do you think?

```
center: {
	label: centerLabel,
	number: centerNumber
}
```

should do it?